src/network/services/force-update.ts

```diff
@@ -0,0 +1,46 @@
+import { Linking, Platform } from 'react-native';
+import DeviceInfo from 'react-native-device-info';
+
+import remoteConfig from '@react-native-firebase/remote-config';
+
+export const isUpdated = (): boolean => {
+  remoteConfig()
+    .setDefaults({
+      minimum_version: '',
+    })
+    .then(() => remoteConfig().fetchAndActivate())
+    .then(fetchedRemotely => {
+      if (fetchedRemotely) {
+        const remoteVersion = remoteConfig().getValue('').asString();
+        const localVersion = DeviceInfo.getVersion();
+        return compareVersions(localVersion, remoteVersion);
+      }
+    });
+  return true;
+};
+
+export const openStore = () => {
+  if (Platform.OS === 'android') {
+    Linking.openURL('');
+  } else {
+    Linking.openURL('');
+  }
+};
+
+function compareVersions(local: string, remote: string): boolean {
+  const localComponents = local.split('.').map(component => parseInt(component));
```

Comment:
The suggestion below will be more descriptive

```suggestion
  const localVersion = local.split('.').map(versionNumber => parseInt(versionNumber, 10));
```

Reply:
Done